pyroma/tests.py

```diff
@@ -159,6 +161,7 @@ def test_minimal(self):
                     "The package's long_description is quite short.",
                     "Your package does not have classifiers data.",
                     "You should specify what Python versions you support.",
```

Comment:
```suggestion
                    "The classifiers should specify what Python versions you support.",
```

This also needs an update to match